### ISSUE TITLE
Plots are not visible in Screen, ScrollView and Carousel

### ISSUE THREAD

**Issue by mostafar** [kivy-garden/garden.graph#7]
When I'm trying to add a graph as a widget to a Screen of a ScreenManager, Ticks and plots will not be visible.

This is the example code:

```
class MyScreen(Screen):
    def __init__(self, **kwargs):
        super(MyScreen, self).__init__(**kwargs)

        self.add_widget(Graph(
            y_ticks_major=10,
            y_grid=True,
            xlabel='X',
            ylabel='Y',
        ))

sm = ScreenManager()
sm.add_widget(MyScreen())

runTouchApp(sm)
```

**Comment by snarfums** [comment]
From above: "line 139 of __init__.py of graph module to:..."
It was line 146 for me. Worked like a charm.

**Comment by rsshruti** [comment]
Hi, I tried to run this program, plot is not visible??
Am I missing something? there is no problem in importing graphs.

Plz help soon!

**Comment by cmac4603** [comment]
Changing `with_stencilbuffer=True` to `with_stencilbuffer=False` in the garden.graph __init__.py module saved my life! I wish I'd have found this page 2 hours ago though!
Thanks @tshirtman & @mostafar! 

P.s. @snarfums: It was line 153 for me, go figure!

**Comment by jcc242** [comment]
Just wanted to add that this is still an ongoing issue. Setting stencilbuffer to False fixed it for me. Thanks!

**Comment by gnarendra89** [comment]
after changing 'with_stencilbuffer=False' , my plot going out of preview(snapshot attached) and if I dont change plot is not shown. am in deadlock :( . am working on real-time graph simulation..
![screenshot from 2016-04-29 19 42 28](https://cloud.githubusercontent.com/assets/12165965/14918739/0e71e188-0e43-11e6-82a1-afabca1adc24.png)

**Comment by khertan** [comment]
This is also required to made it working on iOS

**Comment by rubenbryon** [comment]
Had to search for ages for this, thanks!

**Comment by Kronopt** [comment]
I stumbled into the same problem as @gnarendra89, as you can see from the image:
![drawing_outside](https://cloud.githubusercontent.com/assets/19473287/25866613/efad35ee-34ee-11e7-9458-6c24315b253e.png)

The above graph has a **logarithmic y-axis** and a linear x-axis, while the bottom graph has both x and y-axis in linear scale.
What I find interesting is that for a linear scale axis the plots are drawn correctly, as can be seen in the bottom graph.

I think this shift in graph drawing might be related to this #45.

Does anyone know what might be causing the problem?

**Comment by tito** [comment]
This is the same as https://github.com/kivy/kivy/issues/1578

**Comment by tito** [comment]
This has been fixed in Kivy master (v1.10.1.dev0, git-1114ebf, 20171229)
